### ISSUE TITLE
Replace usage of `focusable` class with `visually-hidden-focusable`

### ISSUE THREAD

**Issue by trackleft** [az-digital/az_quickstart#4653]
## Problem/Motivation
When tabbing, on https://demo.quickstart.arizona.edu/ you used to see a Skip to main content message appear. Now you don't see anything.

this is because we replaced sr-only with visually-hidden but did not replace focusable with visually-hidden-focusable

## Proposed resolution
Description of the proposed solution, the rationale behind it, and workarounds for people who cannot use the patch.

##  Screenshots
If applicable, add screenshots to help explain your problem. (Need a tool to share screenshots or screencasts as URLs? Try [monosnap](https://monosnap.com)
